frontend/package-lock.json

```diff
@@ -18443,6 +18443,16 @@
         "tiny-debounce": "^0.1.1"
       }
     },
+    "react-sortable-hoc": {
```

Comment:
Am I missing something? I'm not seeing a corresponding package.json change in this PR.

Reply:
Mistake